Sources/StreamChat/ChatClient.swift

```diff
@@ -193,7 +193,7 @@ public class _ChatClient<ExtraData: ExtraDataTypes> {
     /// required header auth parameters to make a successful request.
     private var urlSessionConfiguration: URLSessionConfiguration {
         let config = URLSessionConfiguration.default
-        config.waitsForConnectivity = true
+        config.waitsForConnectivity = false
```

Comment:
This change should be covered by a test since it's a requirement for the system to work correctly.

Reply:
Added a test